action/edit.go

```diff
@@ -27,11 +30,21 @@ func (s *Action) Edit(c *cli.Context) error {
 	}
 
 	var content []byte
+	var changed bool
 	if exists {
 		content, err = s.Store.Get(name)
 		if err != nil {
 			return fmt.Errorf("failed to decrypt %s: %v", name, err)
 		}
+	} else if tmpl, found := s.Store.LookupTemplate(name); found {
+		changed = true
+		// load template if it exists
+		content = pwgen.GeneratePassword(24, false)
```

Comment:
Maybe the default `24` should be a const shared across packages or passed down. Aren't we setting that somewhere else too?